frontend/wallet/src/render/components/Footer.re

```diff
@@ -97,10 +97,16 @@ module RightButtons = {
 };
 
 [@react.component]
-let make = (~stakingKey, ~settings) => {
-  <div className=Styles.footer>
-    <StakingSwitch pubKey=stakingKey settings />
-    <ActivityLogButton />
-    <RightButtons pubKeySelected=None settings />
-  </div>;
+let make = () => {
+  let stakingKey = PublicKey.ofStringExn("131243123");
+  let (settingsOrError, _setSettingsOrError) = Hooks.useSettings();
```

Comment:
It's possible I don't understand hooks enough, but I don't think this will work. As implemented, we only want to call `SettingsRenderer.loadSettings()` once and changes via `setSettingsOrError` need to cause rerenders everywhere that use settings (across all components), not just locally.

I'm assuming `useState` behaves like React component state. I think we'll need to mess with the hook to get different `useSettings()` calls to use the same underlying settings structure.

Reply:
useContext?